### DISCUSSION TITLE
Add Clear button on dev tools events listener log

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#21337] [Dashboard] [answered]
currently, when testing for certain events, we can only click Start listening and Stop listening. After having changed things in backend (in my case disable integrations.entities to reduce state changed number) I need to reload the entire view to reset it and need to re enter the event I want to listen to.

A Clear button would be much more user friendly (like we now have on dev tools Templates), and not require the user reloading the view, starting from scratch.
Just clear the logged state changes thus far, and reset the counter:

<img width="914" alt="Scherm­afbeelding 2024-07-08 om 15 27 09" src="https://github.com/home-assistant/frontend/assets/33354141/6b18f2d5-f16b-4ee6-bfb2-63c42429fb78">

<img width="462" alt="Scherm­afbeelding 2024-07-08 om 15 27 20" src="https://github.com/home-assistant/frontend/assets/33354141/3410d3cc-1e0b-4cc4-9633-28b604279af8">

Thanks for considering

**Comment by silamon** [comment, marked as answer]
Implemented with #21353

**Reply by Mariusthvdb** [reply]
Thank you very much indeed!
